src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -47,31 +48,29 @@ export const rejectLiskRequest = (request) => {
 };
 
 // eslint-disable-next-line max-statements
-const RequestSummary = ({ nextStep, history }) => {
+const RequestSummary = ({ nextStep, history, message }) => {
   const { t } = useTranslation();
-  const { getAccountByAddress } = useAccounts();
+  const { getAccountByAddress, accounts } = useAccounts();
   const [currentAccount, setCurrentAccount] = useCurrentAccount();
   const { events } = useEvents();
   const [request, setRequest] = useState(null);
   const [transaction, setTransaction] = useState(null);
   const [senderAccount, setSenderAccount] = useState(null);
   const [errorMessage, setErrorMessage] = useState('');
-  const { sessionRequest } = useSession();
+  const { sessionRequest } = useSession({ isEnabled: !message });
```

Comment:
If we have a message this is not enabled 🤔 ?

Reply:
yhea reason is because we don't need it to be invoked for the message view